pajbot/bot.py

```diff
@@ -813,7 +813,7 @@ def on_pubmsg(self, chatconn, event):
 
         with DBManager.create_session_scope(expire_on_commit=False) as db_session:
             source = User.from_basics(db_session, UserBasics(id, login, name))
-            res = HandlerManager.trigger("on_pubmsg", source=source, message=event.arguments[0])
+            res = HandlerManager.trigger("on_pubmsg", source=source, message=event.arguments[0], tags=tags)
```

Comment:
Seems good to me - please update `pajbot/managers/handler.py:19` to include the newly added parameter

Reply:
🤓 ☝ actually it's line 18